lib/web_ui/lib/src/engine/canvaskit/layer.dart

```diff
@@ -591,7 +591,7 @@ class PlatformViewLayer extends Layer {
   @override
   void preroll(PrerollContext prerollContext, Matrix4 matrix) {
     paintBounds = ui.Rect.fromLTWH(offset.dx, offset.dy, width, height);
-    prerollContext.viewEmbedder!.prerollCompositeEmbeddedView(
+    prerollContext.viewEmbedder?.prerollCompositeEmbeddedView(
```

Comment:
It would be useful for a future reader of this code to leave a comment explaining why it's OK to simply skip embedder view preroll when the viewEmbedder is null.